### PR TITLE
Add button

### PR DESCRIPTION
**Pull request by Krkopiec**

### CODE REVIEW

File: notes.html [hunk 1]
```diff
@@ -8,11 +8,11 @@
...
     <body>
         <div class="container">
-            <div class="row">
+            <div class="row">  
                 <div class="col">
-                    <button type="button" class="btn btn-primary float-right">
+                    <button type="button" id="logout" class="btn btn-primary float-right">
                         <i class="fas fa-sign-out-alt"></i>
-                    </button>
+                    </button>
```
Comment (nkopiec) [inline notes.html]: why you added this space ?

File: notes.html [hunk 2]
```diff
@@ -8,11 +8,11 @@
     </head>
     <body>
         <div class="container">
-            <div class="row">
+            <div class="row">
```
Comment (grkopiec) [inline notes.html:11]: Redundant space on the end of the line

File: scripts/notes.js [hunk 3]
```diff
@@ -5,8 +5,10 @@
     var $ = window.jQuery;
 
     function Notes() {
-        this.getNotes();
-    }
+		this.getNotes();
```
Comment (grkopiec) [inline scripts/notes.js:8]: Broken indentions

File: scripts/notes.js [hunk 4]
```diff
@@ -24,7 +26,13 @@
 				alert('error');
 			}
     	});
-    }
+	}
+
+	Notes.prototype.handlelogout = function() {
+		$('#logout').on('click', function(event) {
+			location.pathname = "/index.html";
```
Comment (grkopiec) [inline scripts/notes.js:33]: Also broken indentions

**Review by grkopiec — CHANGES_REQUESTED** [review]

**Pull request merged**